### PR TITLE
Improve Math test coverage by removing RAD functions, clean up magic numbers

### PR DESCRIPTION
**Pull request by EdNoepel**
Includes change to clean up Fenwick behavior when `findIndexOfSum` rolls past the highest bucket index (lowest price), as reviewed in #410.

### CODE REVIEW

File: src/libraries/Deposits.sol [hunk 1]
```diff
@@ -85,6 +85,9 @@ library Deposits {
         Data storage self,
         uint256 sum_
     ) internal view returns (uint256 sumIndex_) {
+        // Avoid looking for a sum less than the tree size
```
Comment (MikeHathaway) [inline src/libraries/Deposits.sol]: should the comment read "for a sum greater than the tree size"?

**Review by MikeHathaway — APPROVED** [review]
LGTM - just had one question about one of the comments

File: src/libraries/Deposits.sol (hunk 1 above)
Comment (EdNoepel) [reply src/libraries/Deposits.sol]: Yes; good catch.  Fixed in b0918868a95fca7c1d2181badbb0279130b887fd.

**Review by grandizzy — APPROVED** [review]
Lgtm

**Pull request merged**